### DISCUSSION TITLE
how can i use prisma with existing sqlite DB

### DISCUSSION THREAD

**Discussion by MuttakinHasib** [prisma/prisma#7051] [Q&A] [answered]
hi, I want to use **Prisma** with my existing **SQLite DB** . 
how can I create model / migrate to access from client

**Comment by janpio** [comment, marked as answer]
You are looking for Introspection @MuttakinHasib: https://www.prisma.io/docs/concepts/components/introspection

**Reply by MuttakinHasib** [reply]
thanks
